includes/admin/load.php

```diff
@@ -77,7 +77,16 @@ function perflab_render_settings_page() {
  * @param string $hook_suffix The current admin page.
  */
 function perflab_admin_pointer( $hook_suffix ) {
+	$current_user = get_current_user_id();
+
 	if ( ! in_array( $hook_suffix, array( 'index.php', 'plugins.php' ), true ) ) {
+
+		// Do not show on the settings page and dismiss the pointer.
```

Comment:
```suggestion
		// Dismiss the admin pointer once the user reaches the settings screen.
```
Docblock update suggestion